index.html

```diff
@@ -166,4 +168,30 @@ <h1>About <br> Myself</h1>
     </div>
   </section>
 </body>
-</html>
\ No newline at end of file
+
+<section class="formsection">
+  <div class="heading-4">
+    <hx>Contact me</hx>
+  </div>
+  <p>If you have an application you are interested in developing, a feature that you need built or a project that needs coding. I'd love to help with it</p>
+  
+  <form action="https://formspree.io/f/{form_id}" method="post"></form>
```

Comment:
When you indicate the "formspree" server where it says {form_id} you must enter a valid id (registering with that provider)